checks.d/mongo.py

```diff
@@ -355,6 +367,7 @@ class MongoDb(AgentCheck):
         'durability': DURABILITY_METRICS,
         'locks': LOCKS_METRICS,
         'wiredtiger': WIREDTIGER_METRICS,
+        'collection': COLLECTION_METRICS,
```

Comment:
Does it need to be in the default metrics as well ? Looks like it's processed differently, no ?